### DISCUSSION TITLE
保存公共模板后，变量错误

### DISCUSSION THREAD

**Discussion by yalidabuda** [qd-today/qd#376] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/109269051/221086064-eb6f8e5d-00c5-464f-a023-16d2162106f2.png)

群晖docker安装，我在保存公共模板后，变量就自己变成这样了，手动修改也不行，试了两镜像都不行，不知道是哪里的问题。

**Comment by a76yyyy** [comment]
建议提供一下是那个公共模板，使用时有什么影响么？

**Reply by yalidabuda** [reply]
试了两个模板都是这样，在保存前测试没有问题，保存之后再测试就报错了

**Comment by yalidabuda** [comment, marked as answer]
已经解决了，是我没看清密码在上面用户名在下面，填反了。。。。
